contracts/ethereum/contracts/Token.sol

```diff
@@ -583,6 +598,10 @@ contract Token is
         _erc20Out(msg.sender, _recipient, _amount);
     }
 
+    function burnFluidWithoutWithdrawl(uint256 _amount) public {
```

Comment:
Would be good to explain what this is used for in the form of a comment as well as explain why the burning's okay